src/components/quote/README.md

```diff
@@ -0,0 +1,47 @@
+# Quote
+Call prominent attention to a quotation using large text and a citation
+
+## When to use
+Use the quote component when you have a quotation that you would like to display prominently in your page. This could include:
+
+- A sentence or short phrase quoted by an important person
+- A short testimonial from a user about a specific product
+
+## When to use something else
+If the quote is too long to easily read, consider using the HTML `blockquote` element or HTML paragraph styles instead.
```

Comment:
Nice. 💯 This will be an important point to get across. I could see this following the sort of ✅ DO THIS, 🚫 DON'T DO THIS format we've discussed with illustrations of an appropriate usage and a poor usage of the component.